.travis.yml

```diff
@@ -2,7 +2,17 @@ language: generic
 
 script: bash ./deploy.sh
 
+branches:
+  only:
+    - gh-pages
```

Comment:
[`deploy.sh`](https://github.com/w3c/permissions/blob/master/deploy.sh) automatically compiles `master` to `gh-pages`, so we don't have this intermediate step in this repository.